### ISSUE TITLE
feat: Allow setting custom IDs for entries

### ISSUE THREAD

**Issue by ilteriseroglu-ty** [apache/apisix-dashboard#2560]
# Feature request

## Please describe your feature

Right now the dashboard does not allow you to set custom IDs for config entries; which makes manual action & automation in parallel somewhat harder.

## Describe the solution you'd like

An ID entry form box just like entering a name.

## Describe alternatives you've considered

N/A

## Additional context

N/A

**Comment by bzp2010** [comment]
The [ADC](https://github.com/api7/adc) allows you to have fine-grained control over resource allocation and implement gitops, whether this should be implemented on the UI remains to be thought about.

According to #2981, the UI will be directly connected to the Admin API; and as the UI does not seem to be suitable for implementing automation, the gitops tool is an option you should consider. This is currently not on the list of tasks for the next renovation.

If you have any questions, feel free to start a new issue!
